src/lambda/expression.rs

```diff
@@ -38,16 +40,12 @@ pub enum Context {
 
 #[derive(Clone)]
 pub enum Unsafe {
-  Http(
-    RequestTemplate,
-    Option<GroupBy>,
-    Option<Arc<DataLoader<HttpDataLoader, NoCache>>>,
-  ),
+  Http(RequestTemplate, Option<GroupBy>, Option<usize>),
```

Comment:
May be we can take this opportunity to make the fields named here to be consistent with Unsafe::GraphQLEndpoint.